WcaOnRails/.rubocop_todo.yml

```diff
@@ -234,6 +234,7 @@ Style/ClassAndModuleChildren:
     - 'app/controllers/accounts/registrations_controller.rb'
     - 'app/controllers/api/v0/api_controller.rb'
     - 'app/controllers/api/v0/competitions_controller.rb'
+    - 'app/controllers/api/v0/records_controller.rb'
```

Comment:
Damn, why is api_controller and competitions_controller in here? Could we fix those, rather than add another exception? Those files probably violate RuboCop because they were created before we added RuboCop.